### PR TITLE
Removed Moesif from app.js

### LINKED ISSUE #117: Remove Moesif
Remove Moesif. It was added awhile ago, but we don't need it, because we're using AWS CloudWatch.

- [ ] Identify where moesif is used (app.js)
- [ ] Remove these lines of code, and fix it so we don't need them

### PR DESCRIPTION
**Pull request by JakeGerber**
## Reference Issues
Removed Moesif from app.js

## Summary of Change/Fix 
- Verified that changes did not mess up api calls.

## Test Plan
Closes #117

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -129,7 +116,5 @@ module.exports.handler = serverless(app, {binary: ['image/*']});
 
 if (process.env.NODE_ENV == "production") {
   const sentry_wrapper = Sentry.AWSLambda.wrapHandler(serverless(app, {binary: ['image/*']}));
-  const moesif_wrapper = moesif(moesifOptions, sentry_wrapper);
-  module.exports.handler = moesif_wrapper;
```
Comment (ramanxg) [inline app.js:133]: We actually still need this line, but instead we need to let it be `sentry_wrapper` instead.

**Review by ramanxg — CHANGES_REQUESTED** [review]
Thanks! Just one small change I've requested.

**Comment by JakeGerber** [comment]
Fixed the app file line. How would I uncommit the package-lock.json file from the pull request?

**Comment by ramanxg** [comment]
> Fixed the app file line. How would I uncommit the package-lock.json file from the pull request @ramanxg?

You can use checkout the file from master `git checkout origin/master package-lock.json`.

**Comment by JakeGerber** [comment]
> > Fixed the app file line. How would I uncommit the package-lock.json file from the pull request @ramanxg?
> 
> You can checkout the file from master `git checkout origin/master package-lock.json`.

Doesn't do anything. Just says it has updated 0 paths.

**Review by ramanxg — APPROVED** [review]

**Pull request merged**